components/image-builder/pkg/builder/docker.go

```diff
@@ -269,7 +269,7 @@ func (b *DockerBuilder) getBaseImageRef(ctx context.Context, bs *api.BuildSource
 
 func (b *DockerBuilder) getWorkspaceImageRef(ctx context.Context, baseref string, gitpodLayerHash string, allowedAuth allowedAuthFor) (ref string, err error) {
 	//nolint:ineffassign
-	span, ctx := opentracing.StartSpanFromContext(ctx, "getWorkspaceImageRef")
+	span, _ := opentracing.StartSpanFromContext(ctx, "getWorkspaceImageRef")
```

Comment:
Past experience shows that it's better to keep `ctx` around. If it's used in the future tracing doesn't breaking because future us didn't think of adding `ctx` back in. This also explains the `nolint` comment.

Reply:
fixed